Description: enable modules formly from asterisk-addons
 The modules under addons/ are originally from the separate
 asterisk-addons package. As of asterisk 1.8 they are included in the
 main Asterisk distribution but not enabled by default. this patch
 enables them, as it seems valid in Debian.
 .
 format_mp3.c is not enabled, yet, though: the complete source is not
 included. See contrib/scripts/get_mp3_source.sh in the source tree.
Author: Tzafrir Cohen <tzafrir@debian.org>
Applied-Upsteam: no
Last-Update: 2016-03-19

--- a/addons/app_mysql.c
+++ b/addons/app_mysql.c
@@ -34,7 +34,6 @@
 
 /*** MODULEINFO
 	<depend>mysqlclient</depend>
-	<defaultenabled>no</defaultenabled>
 	<support_level>deprecated</support_level>
 	<replacement>func_odbc</replacement>
 	<deprecated_in>1.8</deprecated_in>
--- a/addons/chan_mobile.c
+++ b/addons/chan_mobile.c
@@ -36,7 +36,6 @@
 
 /*** MODULEINFO
 	<depend>bluetooth</depend>
-	<defaultenabled>no</defaultenabled>
 	<support_level>extended</support_level>
  ***/
 
--- a/addons/chan_ooh323.c
+++ b/addons/chan_ooh323.c
@@ -18,7 +18,6 @@
 
 
 /*** MODULEINFO
-	<defaultenabled>no</defaultenabled>
 	<support_level>extended</support_level>
  ***/
 
--- a/addons/res_config_mysql.c
+++ b/addons/res_config_mysql.c
@@ -24,7 +24,6 @@
 
 /*** MODULEINFO
 	<depend>mysqlclient</depend>
-	<defaultenabled>no</defaultenabled>
 	<support_level>extended</support_level>
  ***/
 
--- a/addons/cdr_mysql.c
+++ b/addons/cdr_mysql.c
@@ -35,7 +35,6 @@
 
 /*** MODULEINFO
 	<depend>mysqlclient</depend>
-	<defaultenabled>no</defaultenabled>
 	<support_level>deprecated</support_level>
 	<replacement>cdr_adaptive_odbc</replacement>
 	<deprecated_in>1.8</deprecated_in>
--- a/addons/format_mp3.c
+++ b/addons/format_mp3.c
@@ -28,7 +28,6 @@
  */
 
 /*** MODULEINFO
-	<defaultenabled>no</defaultenabled>
 	<support_level>extended</support_level>
  ***/
 
